packages/web-components/fast-foundation/src/number-field/number-field.ts

```diff
@@ -235,6 +235,10 @@ export class FASTNumberField extends FormAssociatedNumberField {
         this.isUserInput = false;
     }
 
+    public validate(): void {
```

Comment:
Since these functions are public should they have descriptions?

Reply:
Yep good call, updated to inherit docs from FormAssociated